protocol/oracle-manager/contracts/nodes/ReducerNode.sol

```diff
@@ -92,7 +92,10 @@ library ReducerNode {
     function max(
         NodeOutput.Data[] memory parentNodeOutputs
     ) internal pure returns (NodeOutput.Data memory maxPrice) {
-        for (uint256 i = 0; i < parentNodeOutputs.length; i++) {
+        if (parentNodeOutputs.length > 0) {
```

Comment:
would be better to just verify that the number of parent nodes for the reducer node is >0 in the validate function (I believe that is possible right?)